source/linear-algebra/source/04-MX/samples/02.ptx

```diff
@@ -4,139 +4,86 @@
-                                    -4 &amp; 0 \\ -2 &amp; -3 &amp; 1 &amp; 5 &amp; 0 \\ -2 &amp; -1 &amp; 0 &amp; 
-                                    4 &amp; 0 \\ 0 &amp; 0 &amp; -1 &amp; -4 &amp; 1
-                                    \end{array}\right]=\left[\begin{array}{cccc|c} 1 &amp; 0 &amp; 0
-                                    &amp; 0 &amp; 3 \\ 0 &amp; 1 &amp; 0 &amp; 0 &amp; -2 \\ 0 &amp; 0 &amp; 1 &amp;
-                                    0 &amp; -5 \\ 0 &amp; 0 &amp; 0 &amp; 1 &amp; 1 \end{array}\right]</me>
-                        </p>
-                        <p>Therefore its inverse is <m>\left[\begin{array}{cccc} -11 &amp; -8 &amp; 2 &amp; 3
-                              \\ 6 &amp; 4 &amp; -1 &amp; -2 \\ 16 &amp; 12 &amp; -4 &amp; -5 \\ -4
-                              &amp; -3 &amp; 1 &amp; 1 \end{array}\right]</m>. </p>
-                  </solution>
-            </task>
+        <introduction>
+          <p> <me>D = \left[\begin{array}{cccc} 2 &amp; 1 &amp; 3 &amp; 1 \\ -1 &amp; 0 &amp; -2 &amp; -2 \\ -1 &amp; 0 &amp; -1 &amp; -3 \\ 2 &amp; 1 &amp; 5 &amp; 0 \end{array}\right]</me> </p>
+        </introduction>
+          <task>
+            <statement>
+              <p> Explain why this matrix is or is not invertible by discussing its corresponding linear transformation. </p>
+            </statement>
+            <solution>
+              <p> We first calculate <m>\RREF(D)</m>: <me>\mathrm{RREF}\,\left[\begin{array}{cccc} 2 &amp; 1 &amp; 3 &amp; 1 \\ -1 &amp; 0 &amp; -2 &amp; -2 \\ -1 &amp; 0 &amp; -1 &amp; -3 \\ 2 &amp; 1 &amp; 5 &amp; 0 \end{array}\right]=\left[\begin{array}{cccc} 1 &amp; 0 &amp; 0 &amp; 0 \\ 0 &amp; 1 &amp; 0 &amp; 0 \\ 0 &amp; 0 &amp; 1 &amp; 0 \\ 0 &amp; 0 &amp; 0 &amp; 1 \end{array}\right].</me> </p>
+              <p> Since <m>\RREF(D)=I</m>, it follows that <m>D</m> is invertible.   </p>
+            </solution>
+          </task>
+          <task>
+            <statement>
+              <p> If the matrix is invertible, use technology to find its inverse. </p>
+            </statement>
+            <solution>
+              <p>  Its inverse is <m>\left[\begin{array}{cccc} 4 &amp; -7 &amp; 6 &amp; -4 \\ -3 &amp; 9 &amp; -7 &amp; 4 \\ -1 &amp; 1 &amp; -1 &amp; 1 \\ -1 &amp; 2 &amp; -2 &amp; 1 \end{array}\right]</m>.   </p>
+            </solution>
+          </task>
+          <task>
+            <statement>
+              <p> If the matrix is invertible, explain how to find its inverse using techniques that don't require technology. (You may use technology to perform those techniques, as long as they *could* be done without using technology.) </p>
+            </statement>
+            <solution>
+              <p>  To calculate the inverse, we solve the vector equations <m>D\vec{x}=\vec{e}_i</m> for <m>i=1,2,3,4</m>. 
+            These solution vectors then form the columns of the inverse.   </p>
```

Comment:
So this would not be explicility copy-pasted into every solution, we will rewrite this + checkit to ask for a specific column.